api/krusty/configmaps_test.go

```diff
@@ -558,3 +558,23 @@ metadata:
   name: test-k9cc55dfm5
 `)
 }
+
+func TestDataIsSingleQuote(t *testing.T) {
+	th := kusttest_test.MakeHarness(t)
+	th.WriteK(".", `
+configMapGenerator:
+  - name: test
+    literals:
+      - TEST='
+`)
+
+	m := th.Run(".", th.MakeDefaultOptions())
+	th.AssertActualEqualsExpected(
+		m, `apiVersion: v1
+data:
+  TEST: ''''
```

Comment:
I think three single quartation is more correct in this part.
Do you have any problems?

Reply:
> And, your PR failed to lint check. Looks like this fail is not the specific line on your PR, So could you try `rebase master`? Maybe that fixes this failure.

Thanks for the advice. Rebased, although I'm still seeing what appear to be problems with `make lint` locally so I'm not entirely confident I've cleared the issue (it seems like `go fmt` is taking issue with several files not modified by this PR).

Reply:
Thanks!
I understand what happens to this line!